chainercv/extensions/detection/detection_vis_report.py

```diff
@@ -27,179 +22,52 @@ def _check_available():
-        :math:`(R,)`. :math:`R` is the number of bounding boxes in the image.
-        These are integers that correspond to object ID of the dataset.
+        :obj:`gt_label` and :obj:`pred_label` are intenger arrays
+        of shape :math:`(R,)`. Each label indicates the class of
+        the bounding box.
 
-    .. note::
-        All datasets prepared in :mod:`chainercv.datasets` should work
-        out of the box with the default value of :obj:`vis_transform`,
-        which is :obj:`chainercv.transforms.chw_to_pil_image_tuple`.
-        However, if the dataset has been extended by transforms,
-        :obj:`vis_transform` needs to offset some transformations
-        that are applied for correct visualization.
-        For example, when the mean value is subtracted from input images,
-        the mean value needs to be added back in :obj:`vis_transform`.
+        :obj:`pred_score` is a float array of shape :math:`(R,)`.
+        Each score indicates how confident the prediction is.
 
     Args:
-        indices (list of ints or int): List of indices for data to be
-            visualized.
-        target: Link object used for visualization.
-        dataset: Dataset class that produces inputs to :obj:`target`.
-        filename_base (int): Basename for saved image.
-        predict_func (callable): Callable that is used to predict the
-            bounding boxes of the image. This function takes an image stored
-            at the first element of the tuple returned by the dataset with
-            batch dimension added.
-            As an output, this function returns bounding boxes and labels,
-            which have shape :math:`(1, R, 4)` and :math:`(1, R)`
-            respectively. :math:`R` is the number of bounding
-            boxes. Also, the first axis, which is a batch axis, can be removed.
-            Please see description on the step 2 of internal mechanics found
-            above for more detail.
-            If :obj:`predict_func = None`, then :meth:`model.__call__`
-            method will be called.
-        vis_transform (callable): A callable that is used to convert tuple of
-            arrays returned by :obj:`dataset.__getitem__`. This function
-            should return tuple of arrays which can be used for visualization.
-            More detail can be found at the description on the step 3 of
-            internal mechanics found above.
-
+        target (chainercv.links.DetectionLink): Link object used for detection.
+        dataset : Dataset class that produces inputs to :obj:`target`.
+        filename (str): Basename for saved image.
     """
 
     invoke_before_training = False
 
-    def __init__(self, indices, dataset, target,
-                 filename_base='detection', predict_func=None,
-                 vis_transform=_detection_vis_transform):
+    def __init__(
+            self, dataset, target,
+            filename='detection_idx={index}_iter={iteration}.jpg'):
         _check_available()
```

Comment:
Actually, current `filename` was copied from original codes. As you suggested, I will change the order to iteration/index.